README.md

```diff
@@ -258,14 +258,16 @@ class Employee < ActiveRecord::Base
   # specify sequence name
   self.sequence_name = "hr.hr_employee_s"
 
-  # set which DATE columns should be converted to Ruby Date
-  set_date_columns :hired_on, :birth_date_on
+  # set which DATE columns should be converted to Ruby Date using ActiveRecord Attribute API
+  attribute :hired_on, :date
+  attribute :birth_date_on, :date
 
```

Comment:
Starting from Oracle enhanced adapter 1.7,  Oracle `DATE` column is mapped to Rails `Date` by default.  then I am thinking about if these attribute settings should be documented here. I'm going to add this information in the comment.